tests/test_jinja_report.py

```diff
@@ -58,6 +70,59 @@ def test_jinja_report(self):
             '<html><body>\n First rec Second rec\nSome data\n</body></html>'
         )
 
+    @with_transaction()
+    def test_jinja_report_logo(self):
+
+        # TODO: Generate those records with factories when
+        # trytond_factories contain those factories
+        pool = Pool()
+        Attachment = pool.get('ir.attachment')
+        ActionReport = pool.get('ir.action.report')
+        User = pool.get('res.user')
+        Party = pool.get('party.party')
+
+        action_report, = ActionReport.create([{
+            'name': 'Test Report Logo',
+            'report': 'jinja_report/tests/report_template_logo.html',
+            'report_name': 'jinja.test_report_logo',
+            'model': 'test.model',
+            'extension': 'pdf',
+            'template_extension': 'html',
+        }])
+
+        # GIVEN
+        user = User(1)
+        # FIXME: Party factory has customer_payment_term subfactory
+        # that belongs to account module that is not installed
+        # in this test. The same happens with warehouse sufactory
+        # in User factory that depends on stock module.
+        party, = Party.create([{'name': 'Nobody'}])
+        company = trytond_factories.Company.create(party=party)
+        user.company = company
+        user.companies = [company]
+        user.save()
+
+        with open(os.getcwd() + '/tests/logo.png', 'rb') as file:
+            logo = file.read()
+
+        Attachment.create([{
+            'resource': company,
+            'name': 'logo.png',
+            'data': logo,
+        }])
+        Report = pool.get('jinja.test_report_logo', type='report')
+        report_context = Report.get_context([], None, {})
+
+        # WHEN
+        result = Report.render(action_report, report_context)
+
+        # THEN
+        self.assertEqual(
+            result,
+            '<html><body><img src="data:image/png;base64,'
+            f'{b64encode(logo).decode()}"></body></html>'
```

Comment:
The smallest png imaginable, e.g. a 1x1 or 2x2 pixel png, instead of the current tests/logo.png would let you hardcode its base64 encoding so we can avoid this reimplementation in the test suite